data/mods/Aftershock/maps/mapgen/port_augustmoon/port_augustmoon_main.json

```diff
@@ -0,0 +1,329 @@
+      ],
+      "terrain": {
+        ".": "t_hard_vacuum",
+        " ": "t_metal_floor",
+        "!": "t_door_metal_locked",
+        ":": "t_afs_space_ship_hatch_c",
+        "_": "t_grate",
+        "=": "t_afs_augustmoon_diamond_glass",
+        "w": "t_carpet_metal_red",
+        "|": "t_afs_augustmoon_hull_wall",
+        "1": "t_underbrush",
+        "2": "t_nanofab_body",
+        "3": "t_nanofab",
+        "5": "t_elevator_control",
+        "6": "t_elevator"
+      },
+      "furniture": {
+        "0": "f_cryo_pod",
+        "4": "f_mil_comms_sat",
+        "7": "f_autodoc",
+        "8": "f_autodoc_couch",
+        "Ñ": "f_vending_reinforced",
+        "á": "f_neuralnet_inserter"
+      }
+    }
+  },
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ [ "augustmoon_solar_array1" ], [ "augustmoon_solar_array2" ] ],
+    "weight": 200,
+    "object": {
+      "fill_ter": "t_hard_vacuum",
+      "rows": [
+        "  pp                    ",
+        "  pp                    ",
+        "  pp                    ",
+        "  pp                    ",
+        "  pp  pppppppppppppppppp",
+        "  pp  pppppppppppppppppp",
+        "|...........            ",
+        "  pp  pppppppppppppppppp",
+        "  pp  pppppppppppppppppp",
+        "  pp                    ",
+        "  pp                    ",
+        "  pp                    ",
+        "  pp                    ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "  pp                    ",
+        "  pp                    ",
+        "  pp                    ",
+        "  pp                    ",
+        "  pp  pppppppppppppppppp",
+        "  pp  pppppppppppppppppp",
+        "|...........            ",
+        "  pp  pppppppppppppppppp",
+        "  pp  pppppppppppppppppp",
+        "  pp                    ",
+        "  pp                    ",
+        "  pp                    ",
+        "  pp                    ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        ",
+        "                        "
+      ],
+      "palettes": [ "afs_habitat_structure", "afs_habitat_residential_furnishing" ],
+      "terrain": { "1": "t_dirtmound", " ": "t_hard_vacuum", "2": "t_bars", "5": "t_ramp_down_high", "6": "t_ramp_down_low" },
+      "furniture": { "p": "f_solar_unit" }
```

Comment:
Spacecraft typically have radiator arrays also, due to the difficulty of losing heat in space. I don't know if the mod setting changes that or you care about that for this.

Reply:
I do but probably wont care at this specific point in time haha. Its not like the station systems actually do anything.